ogr/services/github.py

```diff
@@ -693,13 +729,31 @@ def _normalize_label_color(color):
             return color[1:]
         return color
 
-    def get_release(self, identifier: int) -> Release:
+    def release_id_from_name(self, name):
+        releases = self.github_repo.get_releases()
+        for release in releases:
+            if release.title == name:
+                return release.id
+        return None
```

Comment:
Please, add type annotation with `Optional` to indicate that we can get `None` as well.

```python
def release_id_from_name(self, name) -> Optional[Release]:
```